extensions/wikia/Email/Controller/WallMessageController.class.php

```diff
@@ -145,6 +145,12 @@ protected static function getEmailSpecificFormFields() {
 					'label' => 'Details',
```

Comment:
I know this isn't part of this PR, but could you update the tooltip for the wallUserName?  Right now it says 'Name of the board, eg <wikiName>.wikia.com/wiki/Board:<wallUserName>' when that's actually the URL for a Forum.  All that needs to change is Board => Message_Wall